ee/clickhouse/queries/actor_base_query.py

```diff
@@ -0,0 +1,70 @@
+import json
+from typing import Any, Dict, List, Optional, Tuple, TypedDict
+
+from rest_framework import serializers
+
+from ee.clickhouse.client import sync_execute
+from posthog.models import Entity, Filter, Team
+
+
+class ActorResponse(TypedDict):
```

Comment:
This really isn't a `Response`, but only a subset of one.

Also there's no real way for the frontend to know given this object if it's dealing with a group or person, which will cause pain.

Rather than a single dict I think this would be much clearer as an union of two:

```
class SerializedPerson(TypedDict):
    type: Literal['person']
    ...

class SerializedGroup(TypedDict):
    type: Literal['group']
   ...

SerializedActor = Union[SerializedPerson, SerializedGroup]
```